modules/references/client/components/CreateReference.component.js

```diff
@@ -150,10 +156,10 @@ export default function CreateReference({ userFrom, userTo }) {
         onSelect={() => {}}
         id="create-reference-tabs"
       >
-        <Tab eventKey={0} title={t('How do you know them')} disabled>
+        <Tab eventKey={0} title={t(`How do you know ${userToName}`)} disabled>
```

Comment:
Also, to note we can't use es6 string interpolation for i18n, e.g.:

```js
t(`How do you know ${userToName}`)
```

Like this, the translation system receives something like "How do you know Nick", which won't match up with a key.

Using i18n interpolation (see [docs](https://www.i18next.com/translation-function/interpolation#basic)) gives us this:

```js
t('How do you know {{name}}', { name: userToName })
```

Now, the translation system (the `t` function) gets passed "How do you know {{name}}", which it can lookup.

(now, some part of it I don't quite understand, as the extraction process did extract the key with a variable `"How do you know $username"`, but not in the format of the other named variables `{{likeThis}}`)